local-tests/setup/session-sigs/get-pkp-session-sigs.ts

```diff
@@ -0,0 +1,45 @@
+import { LitActionResource, LitPKPResource } from '@lit-protocol/auth-helpers';
+import { LitAbility, LitResourceAbilityRequest } from '@lit-protocol/types';
+import { log } from '@lit-protocol/misc';
+import { LitNetwork } from '@lit-protocol/constants';
+import { TinnyEnvironment } from '../tinny-environment';
+import { TinnyPerson } from '../tinny-person';
+
+export const getPkpSessionSigs = async (
+  devEnv: TinnyEnvironment,
+  alice: TinnyPerson,
```

Comment:
Can we rename from `alice` to something more generic, since this might be _any person_ and have a different actual name?